src/wp-includes/rest-api/endpoints/class-wp-rest-font-library-controller.php

```diff
@@ -0,0 +1,394 @@
+						}
+
+						if ( isset( $font_face['downloadFromUrl'] ) && isset( $font_face['uploadedFile'] ) ) {
+							$error_messages[] = sprintf(
+								// translators: 1: font family index, 2: font face index.
+								__( 'Font family [%1$s] Font face [%2$s] should have only one of the downloadFromUrl or uploadedFile properties defined and not both.' ),
+								$family_index,
+								$face_index
+							);
+						}
+
+						if ( isset( $font_face['uploadedFile'] ) ) {
+							if ( ! isset( $files[ $font_face['uploadedFile'] ] ) ) {
+								$error_messages[] = sprintf(
+									// translators: 1: font family index, 2: font face index.
+									__( 'Font family [%1$s] Font face [%2$s] file is not defined in the request files.' ),
+									$family_index,
+									$face_index
+								);
+							}
+						}
+					}
+				}
+			}
+		}
+
+		return $error_messages;
+	}
+
+	/**
+	 * Validate input for the install endpoint.
+	 *
+	 * @since 6.4.0
+	 *
+	 * @param string          $param The font families to install.
+	 * @param WP_REST_Request $request The request object.
+	 * @return true|WP_Error True if the parameter is valid, WP_Error otherwise.
+	 */
+	public function validate_install_font_families( $param, $request ) {
+		$font_families  = json_decode( $param, true );
+		$files          = $request->get_file_params();
+		$error_messages = $this->get_validation_errors( $font_families, $files );
+
+		if ( empty( $error_messages ) ) {
+			return true;
+		}
+
+		return new WP_Error( 'rest_invalid_param', implode( ', ', $error_messages ), array( 'status' => 400 ) );
+	}
+
+	/**
+	 * Gets the schema for the uninstall endpoint.
+	 *
+	 * @since 6.4.0
+	 *
+	 * @return array Schema array.
+	 */
+	public function uninstall_schema() {
+		return array(
+			'fontFamilies' => array(
+				'type'        => 'array',
+				'description' => __( 'The font families to install.' ),
+				'required'    => true,
+				'minItems'    => 1,
+				'items'       => array(
+					'required'   => true,
+					'type'       => 'object',
+					'properties' => array(
+						'slug' => array(
+							'type'        => 'string',
+							'description' => __( 'The font family slug.' ),
+							'required'    => true,
+						),
+					),
+				),
+			),
+		);
+	}
+
+	/**
+	 * Removes font families from the Font Library and all their assets.
+	 *
+	 * @since 6.4.0
+	 *
+	 * @param WP_REST_Request $request Full details about the request.
+	 * @return WP_REST_Response|WP_Error Response object on success, or WP_Error object on failure.
+	 */
+	public function uninstall_fonts( $request ) {
+		$fonts_param = $request->get_param( 'fontFamilies' );
+
+		foreach ( $fonts_param as $font_data ) {
+			$font   = new WP_Font_Family( $font_data );
+			$result = $font->uninstall();
+
+			// If there was an error uninstalling the font, return the error.
+			if ( is_wp_error( $result ) ) {
+				return $result;
+			}
+		}
+
+		return new WP_REST_Response( __( 'Font family uninstalled successfully.' ), 200 );
```

Comment:
This should return an array of remove fonts and boolean.

Reply:
Updated with the changes of this PR https://github.com/WordPress/gutenberg/pull/54829. The output of the install and uninstall endpoints are now standardized for success and errors.

Reply:
Done in https://github.com/WordPress/wordpress-develop/commit/51164435b77deed771f8771c0e7bfb7535b91744 ✅